api/v1beta1/hyperconverged_types.go

```diff
@@ -34,7 +34,7 @@ type HyperConvergedSpec struct {
 	// Important: Run "operator-sdk generate k8s" to regenerate code after modifying this file
 	// Add custom validation using kubebuilder tags: https://book.kubebuilder.io/beyond_basics/generating_crd.html
 
-	// LocalStorageClassName the name of the local storage class.
+	// **[DEPRECATED]** LocalStorageClassName the name of the local storage class.
```

Comment:
While I couldn't find a kubebuilder marker for deprecation, golang does support deprecation of fields/function/types. Please add the golang deprecation:

```suggestion
	// **[DEPRECATED]** LocalStorageClassName the name of the local storage class.
	//
	// deprecated
```

Reply:
Done.
It indeed marks the field with strike-through (at least in GoLand IDE).

Reply:
After reading around, there is a bettergolang format for that:
```suggestion
	// Deprecated: LocalStorageClassName the name of the local storage class.
```